### PR TITLE
Feature/head and sensorring

### PR DESCRIPTION
**Pull request by floweisshardt**
- integrates head and sensorring on one bus
- adapts diagnostics

extends #568

### CODE REVIEW

**Comment by floweisshardt** [comment]
tested on cob4-10: OK

File: cob_bringup/robots/cob4-10.xml [hunk 1]
```diff
@@ -116,21 +116,22 @@
 			<arg name="robot" value="$(arg robot)"/>
 			<arg name="component_name" value="torso"/>
 			<arg name="can_device" value="can0"/>
-			<arg name="cartesian_control" value="true"/>
```
Comment (fmessmer) [inline cob_bringup/robots/cob4-10.xml:119]: could you not just set the value to `false`?

here and wherever it applies...

File: cob_hardware_config/cob4-1/config/diagnostics_analyzers.yaml [hunk 2]
```diff
@@ -104,27 +104,19 @@ analyzers:
...
 #      head:
 #        type: diagnostic_aggregator/GenericAnalyzer
 #        path: Head
-#        contains: 'head/driver'
+#        contains: 'head/'
 #      sensorring:
 #        type: diagnostic_aggregator/GenericAnalyzer
 #        path: Sensorring
-#        contains: 'sensorring/driver'
+#        contains: 'sensorring/'
```
Comment (fmessmer) [inline cob_hardware_config/cob4-1/config/diagnostics_analyzers.yaml:115]: why keep it commented?

File: cob_hardware_config/cob4-1/config/diagnostics_analyzers.yaml [hunk 3]
```diff
@@ -104,27 +104,19 @@ analyzers:
...
-        analyzers:
-          base:
-            type: diagnostic_aggregator/GenericAnalyzer
-            path: drives
-            contains: 'base/driver'
-          base_twist_mux:
-            type: diagnostic_aggregator/GenericAnalyzer
-            path: twist_mux
-            contains: 'twist_mux'
+        contains: 'base/'
```
Comment (fmessmer) [inline cob_hardware_config/cob4-1/config/diagnostics_analyzers.yaml:119]: what happened to the twist_mux diagnostics?
doesn't this `contains` pattern conflict with e.g. base/light?

**Review by fmessmer — COMMENTED** [review]
minor/questions

File: cob_bringup/robots/cob4-10.xml (hunk 1 above)
Comment (mathias-luedtke) [reply cob_bringup/robots/cob4-10.xml:119]: No, the default be kept in a single place.

File: cob_hardware_config/cob4-1/config/diagnostics_analyzers.yaml (hunk 2 above)
Comment (floweisshardt) [reply cob_hardware_config/cob4-1/config/diagnostics_analyzers.yaml:115]: cob4-1 doesn't (yet) have a working sensorring hardware

File: cob_hardware_config/cob4-1/config/diagnostics_analyzers.yaml (hunk 3 above)
Comment (floweisshardt) [reply cob_hardware_config/cob4-1/config/diagnostics_analyzers.yaml:119]: light on base is using `light_base` namespace, so filtering works as expected

**Comment by fmessmer** [comment]
I put a note about the high load problem observed on real hardware caused by the `cartesian_control` [here](https://github.com/ipa320/cob_control/issues/101#issuecomment-274008953)...will work on this as soon as I find the time

**Review by fmessmer — APPROVED** [review]

**Pull request merged**